zeppelin-server/src/test/java/org/apache/zeppelin/AbstractZeppelinIT.java

```diff
@@ -42,8 +42,8 @@
   protected final static Logger LOG = LoggerFactory.getLogger(AbstractZeppelinIT.class);
   protected static final long MIN_IMPLICIT_WAIT = 5;
   protected static final long MAX_IMPLICIT_WAIT = 30;
-  protected static final long MAX_BROWSER_TIMEOUT_SEC = 30;
-  protected static final long MAX_PARAGRAPH_TIMEOUT_SEC = 120;
+  protected static final long MAX_BROWSER_TIMEOUT_SEC = 180;
```

Comment:
I'm not sure why you change it

Reply:
I was trying to fix the `timeout` errors that fails on Travis. It didn't work. I will revert those changes back.